README.md

```diff
@@ -22,21 +22,529 @@ A Spezi framework that provides a common set of SwiftUI views and related functi
+                try await downloadData()
+            }
+            .asyncButtonProcessingStyle(.listRow)
+        }
+        .viewStateAlert(state: $viewState)
+    }
+    
+    private func downloadData() async throws {
+        try await Task.sleep(for: .seconds(2))
+        // Download logic
+    }
+}
+```
+
+#### Tiles and Layout
+
+Create card-like layouts with `SimpleTile`:
+
+```swift
+import SpeziViews
+import SwiftUI
+
+struct BookView: View {
+    var body: some View {
+        SimpleTile(alignment: .leading) {
+            TileHeader(alignment: .leading) {
+                Image(systemName: "book.pages.fill")
+                    .foregroundStyle(.teal)
+                    .font(.largeTitle)
+            } title: {
+                Text("Clean Code")
+            } subheadline: {
+                Text("by Robert C. Martin")
+            }
+        } body: {
+            Text("A handbook of agile software craftsmanship")
+        } footer: {
+            Button("Read More") {
+                // Action
+            }
+            .buttonStyle(.borderedProminent)
+        }
+    }
+}
+```
+
+#### Managed Navigation
+
+Create step-by-step navigation flows:
+
+```swift
+import SpeziViews
+import SwiftUI
+
+struct OnboardingView: View {
+    @State private var path = ManagedNavigationStack.Path()
+    
+    var body: some View {
+        ManagedNavigationStack(path: path) {
+            WelcomeStep()
+            ProfileStep()
+            NotificationStep()
+            CompletionStep()
+        }
+    }
+}
+
+private struct WelcomeStep: View {
+    @Environment(ManagedNavigationStack.Path.self) private var path
+    
+    var body: some View {
+        VStack {
+            Text("Welcome to the App!")
+            Button("Continue") {
+                path.nextStep()
+            }
+        }
+    }
+}
+```
+
+#### Share Sheet
+
+Easily share content with the system share sheet:
+
+```swift
+import SpeziViews
+import SwiftUI
+
+struct ShareView: View {
+    @State private var itemToShare: ShareSheetInput?
+    
+    var body: some View {
+        VStack {
+            Button("Share Text") {
+                itemToShare = ShareSheetInput("Hello World!")
+            }
+            
+            Button("Share Image") {
+                if let image = UIImage(systemName: "star") {
+                    itemToShare = ShareSheetInput(image)
+                }
+            }
+        }
+        .shareSheet(item: $itemToShare)
+    }
+}
+```
+
+#### Skeleton Loading
+
+Add skeleton loading animations:
+
+```swift
+import SpeziViews
+import SwiftUI
+
+struct LoadingView: View {
+    var body: some View {
+        VStack {
+            RoundedRectangle(cornerRadius: 10)
+                .fill(Color(.systemGray4))
+                .frame(height: 100)
+                .skeletonLoading(replicationCount: 5, repeatInterval: 1.5, spacing: 16)
+        }
+        .padding()
+    }
+}
+```
+
+### SpeziPersonalInfo - Personal Information
```

Comment:
I am wondering if this is a key element we want to highlight or if we should rather move that to the DocC landing page for the relevant target and rather add a short link here and link it at the very beginning where we have a short overview?